### ISSUE TITLE
AutoComplete from Primefaces: screen reader output the last menu item "MoreText" as undefined

### ISSUE THREAD

**Issue by prusuteodora** [nvaccess/nvda#19006]
### Brief summary

In the autoComplete component, the last item in the menu that indicates that there are more items is rendered as undefined by the screen readers.

### Steps to reproduce

1. open Chrome(or any other browser)
2. enter this link to the PrimeFaces autocomplete component: https://showcase.primefaces.org/ui/input/autoComplete.xhtml?jfwid=0ad30
3. go to the Max results(5) example and write `a`
4. arrow key to navigate to the `more items available` and see that NVDA is outputing undefined
5. the same happens when it is hovered over with the mouse

<img width="1892" height="862" alt="Image" src="https://github.com/user-attachments/assets/495f9730-e0f9-4152-865c-20b5e2897c3f" />

### Actual behavior

The NVDA writes/announces the item as undefined.

### Expected behavior

The text of the `more items available` should be announced by the NVDA.( in writing and/or sound)

### NVDA type

installed copy

### NVDA version

2025.3

### Windows version

windows 11 enterprise

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I have restarted NVDA with add-ons disabled and the issue still occurs

### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

I have run the COM Registration Fixing Tool and the issue still occurs

**Comment by Emil-18** [comment]
@prusuteodora The website sends The undefined speech to NVDA. They also messed up what NVDA sees as the focus when focus is actually on the more items. list item. You can use object navigation to get to it, and it is reported correctly

**Comment by seanbudd** [comment]
Welcome @prusuteodora 

We are unable to process this issue as it stands.

- To be able to reproduce this issue, we need a code sample. Provide a minimal HTML sample to reproduce this using [codepen](https://codepen.io/). We cannot debug custom frameworks. Ensure the sample only uses plain HTML and JS.
- This is very likely a problem with the implementation of the application/website, as a starting point please submit feedback to the authors to let them know about the problem.

Kind Regards,
NV Access Software Developers

**Comment by melloware** [comment]
@seanbudd @Emil-18 thank you got to the bottom of the issue is the `<li>` did not have an ID to point `aria-activedescendant` to it.  This issue can be closed
